src/cli/cli_network_data.cpp

```diff
@@ -742,7 +767,7 @@ otError NetworkData::OutputBinary(bool aLocal)
  * 44970 5d c000 s 4000 0
  * 44970 01 9a04b000000e10 s 4000 1
  * Contexts:
- * fd00:dead:beef:cafe::/64 1 c
+ * fd00:dead:beef:cafe::/64 1 sc
```

Comment:
I can't comment on the exact line, but just noticed that this block is missing an `@endcode` just before the `@cparam`.  (Because of this, this command isn't appearing in our docs)

It should be:

```
 * Done
 * @endcode
 * @cparam netdata show [@ca{-x}|@ca{rloc16}]
 ```

Reply:
Seems to be an existing issue (also on `main`). Fixed it in this PR.